### PR TITLE
Environment fix for model mgmt

### PR DESCRIPTION
**Pull request by HrishikeshGeedMS**

### CODE REVIEW

File: assets/training/model_management/environments/model-management/context/Dockerfile [hunk 1]
```diff
@@ -18,7 +18,7 @@ COPY requirements.txt .
 RUN pip install -r requirements.txt --no-cache-dir
 
 ## Vulnerability fix
-RUN pip install Pillow==10.3.0 gunicorn==22.0.0 onnx==1.16.0 tqdm==4.66.3 idna==3.7 Jinja2==3.1.4 Werkzeug==3.0.3
+RUN pip install Pillow gunicorn onnx idna tqdm requests
```
Comment (novaturient95) [inline assets/training/model_management/environments/model-management/context/Dockerfile:21]: Jinja2 and werkzeug is removed at the end. Is that fixed in the base image?

File: assets/training/model_management/environments/model-management/context/requirements.txt [hunk 2]
```diff
@@ -13,5 +13,4 @@ azureml-telemetry=={{latest-pypi-version}}
 azureml-automl-dnn-vision=={{latest-pypi-version}}
 pyarrow==14.0.2
 GitPython=={{latest-pypi-version}}
-paramiko=={{latest-pypi-version}}
-tqdm=={{latest-pypi-version}}
\ No newline at end of file
+paramiko=={{latest-pypi-version}}
```
Comment (novaturient95) [inline assets/training/model_management/environments/model-management/context/requirements.txt:16]: Could you please share, how is removing tqdm from here and adding in dockerfile helpful?

**Comment by novaturient95** [comment]
Please add description highlighting what vuln pkgs were there and how it was resolved. Thanks

**Review by novaturient95 — APPROVED** [review]

File: assets/training/model_management/environments/model-management/context/Dockerfile [hunk 3]
```diff
@@ -2,7 +2,7 @@ FROM mcr.microsoft.com/aifx/acpt/stable-ubuntu2004-cu117-py39-torch1131:{{latest
 
 WORKDIR /
 
-RUN apt-get update && apt-get upgrade -y
+RUN apt-get update && apt-get upgrade -y && apt-get install python3-idna -y
```
[the remark this answers is not in the document]
Comment (HrishikeshGeedMS) [reply assets/training/model_management/environments/model-management/context/Dockerfile:5]: For the given vulnerability Ayush as it was ubuntu package - https://ubuntu.com/security/notices/USN-6780-1

**Pull request merged**